src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js

```diff
@@ -20,6 +21,12 @@ import styles from "./ItemSettings.less";
 
 export const ItemSettings = memo(
   function ItemSettings(props) {
+    const showSiteNameInMetaTitle = useSelector(
+      (state) =>
+        state.settings.instance.find(
+          (setting) => setting.key === "show_in_title"
```

Comment:
Selecting on 'key' === 'show_in_title' because it provided the most amount of context.